solr/cores/conf/solrconfig.xml

```diff
@@ -71,10 +71,13 @@
         <lst name="defaults">
             <str name="echoParams">explicit</str>
             <int name="rows">10</int>
-            <str name="df">text</str>
+            <str name="defType">edismax</str>
+            <str name="df">text_exact text_ngram</str>
```

Comment:
If you specify `qf` you don't need to also specify `df`. 

This should probably look more like this:

```xml
 <arr name="qf">
    <str>text_exact^2</str>
    <str>text_ngram</str>
</arr>
```